java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -113,4 +114,24 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> getbit(String key, long offset);
+
+    /**
+     * Perform a bitwise operation between multiple keys (containing string values) and store the
+     * result in the <code>destKey</code>.
+     *
```

Comment:
I think we need to add the warning that all keys in destKey and keys must map to the same hash slot, same for the other docs in this file (the warning does not need to be added to the BaseTransaction file though)